### PR TITLE
feat: Update Watson instrumentation for Node, Python and Java

### PR DESCRIPTION
**Pull request by germanattanasio**
### Node.js
* Update to use node-sdk `3.18.1`.
* Remove deprecated services, unused code (VR shutdown support `api_key`) and some code cleanup
* :new: Add integration tests that start an express server and do REAL api calls to the services to validate that instrumentation and `service-manager` works.
    
### Python
* Update to use python-sdk `2.8.0`.
* Remove deprecated services, unused code and some cleanup
* :new: Add integration tests that start a flask server and do REAL api calls to the services to validate that instrumentation and `service-manager` works.
    image

# Tests
Tests are passing but Travis is not happy
image

### CODE REVIEW

File: generators/service-watson-visual-recognition/templates/go/instrumentation.go [hunk 1]
```diff
@@ -3,40 +3,26 @@ package services
 import (
 	"errors"
 	"github.com/ibm-developer/ibm-cloud-env-golang"
-	// imports might change once the go-sdk is in a public github repo
-	"github.com/watson-developer-cloud/golang-sdk/visualRecognitionV3"
-	watson "github.com/watson-developer-cloud/golang-sdk"
+	"github.com/watson-developer-cloud/go-sdk/core"
+  "github.com/watson-developer-cloud/go-sdk/visualrecognitionv3"
```
Comment (jkingoliver) [inline generators/service-watson-visual-recognition/templates/go/instrumentation.go]: @germanattanasio will you check formatting in this file please?

File: generators/language-swift-kitura/bluemix-label-mappings.json [hunk 2]
```diff
@@ -2,7 +2,6 @@
   "cloudant": "cloudant",
   "objectStorage": "object_storage",
   "appid": "appid",
-  "conversation": "watson_assistant",
```
Comment (jkingoliver) [inline generators/language-swift-kitura/bluemix-label-mappings.json]: why is this mapping removed?

Comment (germanattanasio) [reply generators/language-swift-kitura/bluemix-label-mappings.json]: The service was added as `watson_assistant` and therefore generating two sets of credentials. While the service name changed, the bluemix label didn't so we should keep using `conversation`.
In this PR i'm moving the templates to the conversation service. If you look at the state of `develop` you will find swift and go in `watson-assistant` and Node Python and Java in `conversation`.

I will be updating the other languages in the next PR.

**Comment by jkingoliver** [comment]
@germanattanasio will you squash your commits please?

File: generators/language-swift-kitura/bluemix-label-mappings.json (hunk 2 above)
Comment (christiancompton) [reply generators/language-swift-kitura/bluemix-label-mappings.json]: I am not sure about the implications on this for Go,  I think the changes here can proceed.

**Comment by christiancompton** [comment]
@germanattanasio Please remove the Swift refactoring from above - the credentials are done differently from all other lanuggaes due to the lack of a JSON path library and adequate support - as result the code is far more brittle, and the naming must mach exactly.

File: generators/language-swift-kitura/bluemix-label-mappings.json (hunk 2 above)
Comment (germanattanasio) [reply generators/language-swift-kitura/bluemix-label-mappings.json]: I think the problem is `Swift credential reading library and generator-swiftserver both pass `assistant` inputs`.

I will rollback those changes but please don't take any more decision that involves the watson user experience without consulting with the right team. The fact that we now have `watson_assistant` and `watson_conversation` is a HORRIBLE experience.


I'm just going to limit my contributions to update the existing node, python and java template.

**Comment by germanattanasio** [comment]
> @germanattanasio will you squash your commits, please?
@jkingoliver The commits are separated based on the work I did if you want you can squash the PR into one commit.

**Comment by germanattanasio** [comment]
I just added the tests and changes to update the Java templates too

File: generators/service-watson-conversation/templates/swift/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+## Watson Assistant
```
Comment (christiancompton) [inline generators/service-watson-conversation/templates/swift/README.md]: Delete.

File: generators/service-watson-conversation/templates/swift/instrumentation.swift [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import Foundation
```
Comment (christiancompton) [inline generators/service-watson-conversation/templates/swift/instrumentation.swift]: Delete.

File: generators/service-watson-conversation/templates/swift/README.md (hunk 3 above)
Comment (christiancompton) [reply generators/service-watson-conversation/templates/swift/README.md]: Or move to the `watson-assitant` dir if applicable.

File: generators/service-watson-conversation/templates/swift/meta.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (christiancompton) [inline generators/service-watson-conversation/templates/swift/meta.json]: Delete.

File: generators/service-watson-text-to-speech/templates/java-liberty/README.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# Watson Text to Speech
+
+
+## Documentation
+
+ * API Reference: https://cloud.ibm.com/apidocs/text-to-speech?code=swift
```
Comment (christiancompton) [inline generators/service-watson-text-to-speech/templates/java-liberty/README.md]: shouldn't this be Java on the anchor?

File: generators/service-watson-text-to-speech/templates/java-spring/README.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+# Watson Text to Speech
+
+
+## Documentation
+
+ * API Reference: https://cloud.ibm.com/apidocs/text-to-speech?code=swift
```
Comment (christiancompton) [inline generators/service-watson-text-to-speech/templates/java-spring/README.md]: java?

**Pull request merged**